nexus-sdk-js/src/__mocks__/helpers.ts

```diff
@@ -374,3 +374,25 @@ export const mockElasticSearchViewQueryResponse: ElasticSearchViewQueryResponse
   timed_out: false,
   took: 8,
 };
+
+export const mockFileResponse: ElasticSearchViewQueryResponse = {
+  "@context": "https://bluebrain.github.io/nexus/contexts/resource.json",
+  "@id": "schema:b9fa7dba-1da4-4cb6-99f8-661779ebbc3c",
+  "@type": "File",
+  "_bytes": 89859,
+  "_digest": {
+      "_algorithm": "SHA-256",
+      "_value": "f3ffad24d3aaad96ae533201a0dc8b6cc4299dc7889256b880b36b0f2ffca9c1"
+  },
+  "_filename": "Screenshot 2019-01-16 at 10.45.06.png",
+  "_mediaType": "image/png",
+  "_self": "http://dev.nexus.ocp.bbp.epfl.ch/v1/files/broman/first/base:b9fa7dba-1da4-4cb6-99f8-661779ebbc3c",
```

Comment:
exposing internal urls... they should be replaced by dummy ones or at least the production one